lib/suspenders/generators/app_generator.rb

```diff
@@ -239,7 +239,7 @@ def remove_routes_comment_lines
 
     def outro
       say 'Congratulations! You just pulled our suspenders.'
-      say "Remember to run 'rails generate airbrake' with your API key."
+      honeybadger_outro
```

Comment:
I'd rather that be called `say_honeybadger_outro` or else rephased as `say honeybadger_outro` to communicate what it's doing.